privilege/privilege.go

```diff
@@ -57,10 +57,23 @@ type Manager interface {
 	// RequestDynamicVerificationWithUser verifies a DYNAMIC privilege for a specific user.
 	RequestDynamicVerificationWithUser(privName string, grantable bool, user *auth.UserIdentity) bool
 
+	// IsAccountAutoLockEnabled verifies whether the account has enabled Failed-Login Tracking and Temporary Account Locking.
+	IsAccountAutoLockEnabled(user string, host string) bool
+
+	// BuildPasswordLockingJSON builds PasswordLocking JSON string.
+	BuildPasswordLockingJSON(failedLoginAttempts int64,
```

Comment:
Is `BuildPasswordLockingJSON` and `BuildSuccessPasswordLockingJSON` required for `Manager` interface? Could we turn it to a package function instead of interface function?